scripts/scenarios/cloud6/cloud7-upgrade-non-disruptive-ceph.yml

```diff
@@ -93,6 +93,7 @@ proposals:
         admin_keyring: "/etc/ceph/ceph.client.admin.keyring"
         pool: volumes
         user: cinder
+        secret_uuid: 75ffb452-2478-4c4c-abcc-1be39cb6325a
```

Comment:
I think it should work as that is the default value when you deploy the rados backend for cinder from the UI.

Do we have tome to we test it with that value? I rather have the default value than a random UUID which can get confusing in the future, but I dont want to block QA here.